pegasus/src/homepage.rb

```diff
@@ -555,14 +555,15 @@ def self.show_courses_banner(request)
 
   def self.get_dance_stars
     stars = [
-      "Katy Perry", "Lil Nas X (ft. Billy Ray Cyrus)", "Jonas Brothers", "Panic! At The Disco",
-      "Shawn Mendes", "Nicki Minaj", "KIDZ BOP", "Pedro Capó", "Francesco Gabbani", "Sia",
-      "A-ha", "Ariana Grande", "Avicii and Aloe Blacc", "Calvin Harris",
-      "Carly Rae Jepsen", "Ciara", "Coldplay", "Ed Sheeran", "Imagine Dragons",
+      "Beyoncé", "Harry Styles", "Lizzo", "Post Malone", "Disney\'s \"Encanto\"", "Nicky Youre",
+      "Katy Perry", "Lil Nas X", "Jonas Brothers", "Panic! At The Disco",
+      "Shawn Mendes", "Nicki Minaj", "Pedro Capó", "Francesco Gabbani", "Sia",
+      "Ariana Grande", "Avicii and Aloe Blacc", "Calvin Harris",
+      "Carly Rae Jepsen", "Coldplay", "Ed Sheeran", "Imagine Dragons",
       "J Balvin and Willy William", "Justin Bieber", "Keith Urban", "Lady Gaga",
       "Los del Río", "Madonna", "Mark Ronson (ft. Bruno Mars)", "MC Hammer",
       "Miley Cyrus", "Selena Gomez", "The Weeknd", "Yolanda Be Cool"
     ]
-    DCDO.get("hoc2019_dance_stars", stars)
+    return stars
```

Comment:
@megcrenshaw can you take a quick peek at this? I'm not super familiar w/ Ruby and want to make sure this looks ok! I was getting an linting error after removing the DCDO flag from the convo above and this fixed it.

Reply:
Ah, there are some weird things happening here –– 
1) Ruby methods return whatever the last line is, unless there's a return statement earlier (see https://launchschool.com/books/ruby/read/methods). So you could make the whole method be 
```
def self.get_dance_stars
    [
      "Katy Perry", "Lil Nas X (ft. Billy Ray Cyrus)", "Jonas Brothers", "Panic! At The Disco",
      "Shawn Mendes", "Nicki Minaj", "KIDZ BOP", "Pedro Capó", "Francesco Gabbani", "Sia",
      "A-ha", "Ariana Grande", "Avicii and Aloe Blacc", "Calvin Harris",
      "Carly Rae Jepsen", "Ciara", "Coldplay", "Ed Sheeran", "Imagine Dragons",
      "J Balvin and Willy William", "Justin Bieber", "Keith Urban", "Lady Gaga",
      "Los del Río", "Madonna", "Mark Ronson (ft. Bruno Mars)", "MC Hammer",
      "Miley Cyrus", "Selena Gomez", "The Weeknd", "Yolanda Be Cool"
    ]
  end
```
This is a very weird Ruby thing.
2) The other weird thing happening here is that `DCDO.get("hoc2019_dance_stars", stars)` returns `stars`, which makes the whole method return the `stars` array.

Since now the variable called `stars` isn't used anywhere, you could just have the array itself (as shown above).

Does that make sense?

Reply:
Ah ha! Yes that makes sense, thank you — just updated 👍